lua/entities/gmod_wire_egp/lib/objects/3dtracker.lua

```diff
@@ -91,7 +88,7 @@ function Obj:Transmit()
 	net.WriteFloat( self.target_y )
 	net.WriteFloat( self.target_z )
 	net.WriteEntity( self.parententity )
-	net.WriteInt((self.angle%360)*64, 16)
+	net.WriteInt(self.angle * 64, 16)
```

Comment:
Why remove this mod? Is it modded elsewhere?

Reply:
It's supposed to be modded only when angle is modified, but I forgot to add that in the baseclass `SetPos`.